docs_src/microservices/core/command/Ch-Command.md

```diff
@@ -90,12 +90,384 @@ The two following High Level Diagrams show:
 
 ## Configuration Properties
 
-Please refer to the general [Common Configuration documentation](../../configuration/CommonConfiguration.md) for configuration properties common to all services. Core Command no longer has any additional settings.
+Please refer to the general [Common Configuration documentation](../../configuration/CommonConfiguration.md) for configuration properties common to all services.
+Below are only the additional settings and sections that are not common to all EdgeX Services.
+
+!!! edgey "EdgeX 2.3"
+    `RequireMessageBus` and `MessageQueue` configurations were added in EdgeX 2.3 for North-South Messaging
+
+=== "RequireMessageBus"
```

Comment:
The Device Services (device-sdk-c, at least) have "UseMessageBus" to determine whether to publish Event updates on the Message Bus. 
That's a confusingly similar name to "RequireMessageBus", with a different meaning. Probably need to add something about "UseMessageBus" too, to keep them apart in understanding.

Reply:
@lenny-intel do you think we need to update this config to `UseMessageBus` ?

Reply:
OK, makes sense.